bbb-common-web/src/main/java/org/bigbluebutton/api/model/validator/ContentTypeValidator.java

```diff
@@ -31,7 +32,13 @@ public boolean isValid(Request request, ConstraintValidatorContext context) {
         if (requestBodyPresent) {
             if (contentType == null || contentTypeHeader == null) return false;
             else {
-                return request.getSupportedContentTypes().contains(contentType);
+                ContentType c = ContentType.parse(contentType);
+                String mimeType = c.getMimeType();
+                for (Object o: request.getSupportedContentTypes()) {
+                    String supportedContentType = (String) o;
+                    if (mimeType.equalsIgnoreCase(supportedContentType)) return true;
```

Comment:
Was curious if the `equalsIgnoreCase` is needed here; looks like it is; the `.parse` method does not normalize the mime type to lowercase from my reading of https://github.com/apache/httpcomponents-core/blob/4.4.x/httpcore/src/main/java/org/apache/http/entity/ContentType.java